__tests__/main.test.ts

```diff
@@ -144,6 +145,7 @@ describe('run', () => {
   describe('when the action is triggered by a different actor', () => {
     beforeEach(() => {
       process.env.GITHUB_ACTOR = 'classic-rando'
```

Comment:
I think this should be dependabot[bot]?
> Now when a re-run occurs the GITHUB_ACTOR stays as dependabot[bot] and the GITHUB_TRIGGERING_ACTOR indicates the user that triggered the re-run.

```suggestion
      process.env.GITHUB_ACTOR = 'dependabot[bot]'
```

Reply:
That's covered in the next tests. This test would cover the case where this action has been added to a custom workflow and the `GITHUB_ACTOR` would be the user that initially triggers it.